code/game/machinery/machinery.dm

```diff
@@ -120,19 +120,27 @@ Class Procs:
 	var/speed_process = 0 // Process as fast as possible?
 
 /obj/machinery/initialize()
-	addAtProcessing()
+//	addAtProcessing() //Testing this in new conveyors
 	. = ..()
 	power_change()
 
 /obj/machinery/proc/addAtProcessing()
 	if(use_power)
 		myArea = get_area_master(src)
 
+/*	machines += src
+	if(!speed_process)
+		machine_processing += src
+	else
+		fast_processing += src */ //Also testing in NEW conveyors
+
+/obj/machinery/new() //new for trying to get the conveyors working with machinery, not world
 	machines += src
 	if(!speed_process)
 		machine_processing += src
 	else
 		fast_processing += src
+	. = ..()
 

```

Comment:
Leave the "add to processing" lines alone, and put them back - otherwise, a lot of things will catch on fire